app/components/Card/CardBack.tsx

```diff
@@ -24,21 +25,23 @@ const CardBack: React.FC<CardBackProps> = ({
   setIsFlipped,
   name,
   description,
+  path,
 }) => {
+  const style =
+    path === "/" ? responsiveValues.carousel : responsiveValues.machinesPage;
```

Comment:
This is a really smart way to style for two different pages/layouts in one component, I wonder if I can use this logic this for the quote request form across different pages!